.github/workflows/helm-test.yaml

```diff
@@ -1,15 +1,11 @@
 name: helm-test
```

Comment:
Do we still need this workflow? It's a duplicate of what we have in helm-release, can we remove it?

Reply:
I think we do because we need to know that the helm chart is ok before you can merge the PR. The other pipelines only runs on push to main. We only really need the tests duplicated into the other pipeline now because linting is turned off and because if someone pushed straight to main without a PR somehow, you'd not have any tests run before the release

Reply:
Good point, let's keep it.

Reply:
Ideally we want the linting on of course but with the bug when the name has a colon we'll have to make do with the duplicate section for now, or get rid of the colon on those resource names.

Reply:
We are following the [k8s naming convention](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#discovery-roles) adding the prefix `kyveno:` to each ClusterRole, I would say let's keep the names and use helm-tests to capture the invalid cases.